### ISSUE TITLE
Environment Variables

### ISSUE THREAD

**Issue by tmrobert8** [openhab/openhab-distro#814]
Looking at setenv - looks like the following variables (directory related) are used:
1. OPENHAB_HOME - home of openhab
2. OPENHAB_CONF - configuration [defaults to OPENHAB_HOME/conf]
3. OPENHAB_RUNTIME - the runtime [defaults to OPENHAB_HOME/runtime]
4. OPENHAB_USERDATA - the userdata [defaults to OPENHAB_HOME/userdata]

In addition - the backup scripts use:
1. OPENHAB_BACKUPS - the backup locations [defaults to OPENHAB_HOME/backups]

What is suspiciously missing is an environment variable to define the location of the "addons" directory.  All the scripts (and I'm assuming the system) assumes the addons will always be "OPENHAB_HOME/addons".

Is there any chance we can implement a OPENHAB_ADDONS environment variable that will specify the location of the addons directory?  I don't know if there is a technical reason why this isn't possible or not.

Note: kinda would like to hear whether this is possible or not - I'm writing the installer and redoing the windows scripts and need to know whether I should include a OPENHAB_ADDONS or not (which I've acutally done - won't affect anything since nothing uses it)..

**Comment by BClark09** [comment]
You're right that it is assumed that the addons will always be placed in "$OPENHAB_HOME/addons". Whilst the addons folder isn't currently configurable in openHAB, it is possible to change.

[userdata/etc/org.apache.felix.fileinstall-deploy.cfg](https://github.com/openhab/openhab-distro/blob/master/distributions/openhab/src/main/resources/userdata/etc/org.apache.felix.fileinstall-deploy.cfg) would need to be modified to use `{openhab.addons}` and then this can be set using `oh_dir_layout` and `setenv`. @kaikreuzer do you see any issue in this? I can make a PR if not.

**Comment by kaikreuzer** [comment]
I do not see any issues - if there are any, you will find them out :-)
